thereminderbot/src/main/java/org/thereminderbot/repository/RemindRepository.java

```diff
@@ -5,29 +5,59 @@
 
 import java.util.ArrayList;
 
-/**
- * Репозиторий заметок.
- */
 public class RemindRepository {
-    private ArrayList<Remind> reminds;
+    private final ArrayList<Remind> reminds;
+
+    public ArrayList<Remind> getAll() {
+        return new ArrayList<>(reminds);
+    }
 
     public RemindRepository() {
-        //TODO
-        return;
+        // Инициализация пустого хранилища
+        this.reminds = new ArrayList<>();
     }
 
-    public ArrayList<Remind> getRemindById(long id) {
-        //TODO
-        return null;
+    /**
+     * Получить все напоминания по userId
+     */
```

Comment:
Добавь описание аргумента

Reply:
@danone05 
Закрой тред сам, плз
```suggestion
    /**
     * Получить одно напоминание по его идентификатору.
     * @param Id напоминания.
     * @return объект {Remind}, если найден; иначе {null}.
     */
```